### PR TITLE
Support viewing user flows for entries created with latest release

### PR DESCRIPTION
**Pull request by AbdiTolesa**
This PR is aimed at backward compatibility that allows the user journey of new entries created with the new version of the user journey add on to be viewed using the `[frm-show-entry]` shortcode. The attached screenshot shows an example of such scenario after this update.

image

### CODE REVIEW

File: formidable-user-tracking.php [hunk 1]
```diff
@@ -31,6 +32,7 @@ public static function load_hooks() {
 		}
 		add_action( 'frm_after_create_entry', 'Frm_User_Tracking::insert_tracking_into_entry', 15 );
 		add_action( 'admin_init', 'Frm_User_Tracking::include_auto_updater', 1 );
+		add_filter( 'frm_user_info_referrer', 'Frm_User_Tracking::append_user_journey', 10, 2 );
```
Comment (Crabcyborg) [inline formidable-user-tracking.php:35]: @AbdiTolesa isn't `Frm_User_Tracking` deprecated? Would it make more sense to just include this with https://github.com/Strategy11/formidable-user-tracking/pull/6 and use the AppHelper or something that will actually exist in next release? I don't imagine this update is actually going to be released separately, right?

File: user-tracking-view.php [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+				<td colspan="3"><?php echo esc_html( gmdate( $date_format, strtotime( $key ) ) ); ?></td>
+			</tr>
+				<?php
+			}
+
+			if ( ! is_array( $pages ) ) {
+				continue;
+			}
+
+			foreach ( $pages as $page_ ) {
```
Comment (Crabcyborg) [inline user-tracking-view.php:25]: Ideally we can avoid names with trailing or leading underscores like `$page_`. It isn't really a very standard variable name. If there is another variable called `$page` the best thing to do is give them both longer more descriptive names. These vague names just make it harder to follow and read.

File: formidable-user-tracking.php (hunk 1 above)
Comment (AbdiTolesa) [reply formidable-user-tracking.php:35]: @Crabcyborg I was assuming this as an update to v1 to allow users be able to view entries created with latest version of the addon using older versions. Steph added this to v1.01 milestone and that's why I thought so. In fact, similar update is required for v2 of the add on as well and I have a plan to incorporate it in the rock update.

Comment (Crabcyborg) [reply formidable-user-tracking.php:35]: I don't believe there will be any new v1.x release. I also don't see any old milestone.

I think all updates will happen at once with the other update.

Comment (AbdiTolesa) [reply formidable-user-tracking.php:35]: @Crabcyborg I might prpbably be misguided by the information in the issue page then. Here is what I see from there:
Screenshot_20220927-231253_Samsung Internet

Comment (Crabcyborg) [reply formidable-user-tracking.php:35]: As far as I can tell the molestone was just renamed to v2.

There's just going to be one new release.

File: user-tracking-view.php (hunk 2 above)
Comment (AbdiTolesa) [reply user-tracking-view.php:25]: @Crabcyborg It makes sense to do that indeed, thank you!

File: formidable-user-tracking.php (hunk 1 above)
Comment (AbdiTolesa) [reply formidable-user-tracking.php:35]: @Crabcyborg Sorry for not realizing that earlier myself! I just moved the same update to the rock update PR.

**Pull request closed without merging**